app/javascript/components/widget-transition-controls.jsx

```diff
@@ -162,7 +162,7 @@ WidgetTransitionControls.propTypes = {
   cityName: PropTypes.string.isRequired,
   location: PropTypes.shape({
     widgets: PropTypes.shape({
-      enabled: PropTypes.array.isRequired,
+      enabled: PropTypes.arrayOf.isRequired,
```

Comment:
Lets split out the lint changes to a commit that precedes to eslint upgrade.

Reply:
There's actually a lot of these changes, but they are in their own commit now!